bootstrap/043_routines.sql

```diff
@@ -0,0 +1,7 @@
+
+create or replace function admin.list_routines()
+    returns object
+as
+$$
+    internal.wrapper_list_routines(object_construct())
```

Comment:
sorry, I missed this. We need to return a table, not an object.

Reply:
can you pls keep the columns consistent with `show sundeck routines;` ?
![Screenshot 2024-03-01 at 6 19 10 PM](https://github.com/sundeck-io/OpsCenter/assets/4161430/1c2796ae-a439-4b7a-9c95-2b0f7a7ae970)

Reply:
fixed it and verified against dev

![Screenshot 2024-03-04 at 3 39 34 PM](https://github.com/sundeck-io/OpsCenter/assets/42910861/bcaf2be6-38b4-411a-ab67-98e6ff438297)